erc190/package.py

```diff
@@ -30,6 +31,10 @@ def __init__(self, lockfile):
             validate(package_data, schema_data)
         except jsonValidationError:
             raise ValidationError
+
+        if package_data['build_dependencies']:
```

Comment:
this will throw a `KeyError` if the `build_dependencies` key is not present (which is allowed).  Can you pull in the `owned` package/lockfile example as a test case for this.